flight/views.py

```diff
@@ -1,3 +1,15 @@
-from django.shortcuts import render
+import json
+from django.views  import View
+from django.http   import JsonResponse
+from flight.models import Service
 
-# Create your views here.
+class ServiceBundleView(View):
+    def get(self, request):
+        services = Service.objects.all()
+        service  = [
+                {
+                    'name' : service.name,
+                    'image' : service.image_url
+                    } for service in services]
```

Comment:
```suggestion
        services = Service.objects.all()
        service  = [
                {
                    'name' : service.name,
                    'image' : service.image_url
                } for service in services]
```
같은 레벨의 괄호는 같은 indentation을 갖도록 하는 방법을 추천드립니다.

Reply:
수정하겠습니다!